coil-base/src/main/java/coil/memory/RequestService.kt

```diff
@@ -158,18 +84,18 @@ internal class RequestService(
      */
     @WorkerThread
     private fun isConfigValidForHardwareAllocation(request: ImageRequest, size: Size): Boolean {
-        return isConfigValidForHardware(request, request.bitmapConfig ?: defaults.bitmapConfig) &&
+        return isConfigValidForHardware(request, request.bitmapConfig) &&
             hardwareBitmapService.allowHardware(size, logger)
     }
 
     /** Return true if [ImageRequest.bitmapConfig] is valid given its [Transformation]s. */
     private fun isConfigValidForTransformations(request: ImageRequest): Boolean {
-        return request.transformations.isEmpty() || (request.bitmapConfig ?: defaults.bitmapConfig) in VALID_TRANSFORMATION_CONFIGS
+        return request.transformations.isEmpty() || request.bitmapConfig in VALID_TRANSFORMATION_CONFIGS
     }
 
     companion object {
-        /** A whitelist of valid bitmap configs for the input and output bitmaps of [Transformation.transform]. */
-        @JvmField internal val VALID_TRANSFORMATION_CONFIGS = if (SDK_INT >= 26) {
+        /** An allowlist of valid bitmap configs for the input and output bitmaps of [Transformation.transform]. */
+        @JvmField val VALID_TRANSFORMATION_CONFIGS = if (SDK_INT >= 26) {
```

Comment:
Purposefully made public?

Reply:
It's actually still `internal` since `RequestService` is internal.

Reply:
Oh, my bad!